includes/definitions/discovery/linux.yaml

```diff
@@ -35,27 +35,27 @@ modules:
                     descr: 'Virtual disks'
                 -
                     oid: adapterPropertiesTable
-                    value: pdDiskPredFailureCount
-                    num_oid: '.1.3.6.1.4.1.3582.4.1.4.1.2.1.19.{{ $index }}'
+                    value: vdOfflineCount
```

Comment:
Hmm, the description of this sensor is "Virtual disk failure".
A  disk could be offline for multiple reasons.
Perhaps you should define a new sensor instead?

Reply:
In this context what they refer to as a virtual disk is a RAID array.  This happens if a RAID array has failed (multi disk failure, or single disk failure for RAID0).  I'm not sure what other conditions can trigger the RAID array to go offline, but this can be changed if there's a better OID in the MIB to use for a disk failure.  I can say that the physical disk predictive failure is not the right OID.

Reply:
Regarding 0 always alerting, I am quite sure that I originally tested using 0 when I originally found the issue, and have been running this as a manual patch.  I have just tested again and using 0 does come into the database correctly.  Either I am wrong for the current state of LibreNMS, and I will update these values to 0.

I can confirm that setting this to 0 does not alarm all the time.  It appears that the alerts only happen when you exceed the high value, and not reach it.  See below where the web UI shows the sensors as green when they are the same as the high value:
![image](https://github.com/librenms/librenms/assets/16013255/290f0fd7-3389-4b49-9e5f-551bdd666f38)

Reply:
> Hmm, the description of this sensor is "Virtual disk failure". A disk could be offline for multiple reasons. Perhaps you should define a new sensor instead?

Any update on this?  I am very sure that virtual disk offline is indicative of a RAID array failure (e.g. 2 disk RAID5).  If there are other reasons why a virtual disk is offline, it would be an unusual deployment, and the admin of the system can adjust the threshold manually.

The reason I have modified the existing sensor instead of defining a new one is because the OID that is currently configured on the "Virtual disk failure" sensor in LibreNMS is the Phyiscal Disk predictive failure OID, which is clearly not correct.  The Phyiscal Disk predictive failure OID is also used correctly on the "Critical disks" sensor, so the Virtual disk failure" sensor is currently a duplicate of this.

Reply:
I'm happy to just adjust all the alarm values to 0 so they alarm when 1 disk fails, and I can change the code to remove the "Virtual disk failure" sensor if there are still questions on the usefulness.

As it currently stands these sensors do not alert until 2 disks fail, and the "Virtual disk failure" sensor looks at predictive failures.  I am quite sure that the vdoffline counter is at worst case the best OID for detecting RAID array failures, and am certain that having 2 sensors looking at the same OID is pointless.

Reply:
How about this - I've swapped back to the original sensor OID, but have updated the MIB reference to match and also corrected the description to match the OID (degraded array vs failed array).

If the above change is acceptable, what are the thoughts on adding another sensor for a RAID array failure?

Reply:
Here is the location of code that determines the status shown on the web page, where sensors must be > the limit to change colour.  This is what I was seeing where a single failure still showed as green in the UI:
https://github.com/librenms/librenms/blob/a6809e4dcfaa303b82e970f131a4c489f062b07e/includes/html/functions.inc.php#L1085